### PR TITLE
Testing Pull Request

### PR DESCRIPTION
**Pull request by Rayal-Khatri**
Added a new feature to see your City name you have entered

### CODE REVIEW

File: Django/weatherdetector/templates/index.html [hunk 1]
```diff
@@ -35,7 +35,7 @@
   </center>
   <div class="row">
     <div class="col-md-6 col-md-offset-3">
-      <h1>city</h1>
```
Comment (Rayal-Khatri) [inline Django/weatherdetector/templates/index.html:38]: This is Hardcoded to display City

File: Django/weatherdetector/templates/index.html [hunk 2]
```diff
@@ -35,7 +35,7 @@
...
+      <h1>{{city}}</h1>
```
Comment (Rayal-Khatri) [inline Django/weatherdetector/templates/index.html:38]: This takes an input from user and displays the name of the City they want to see

File: Django/weatherdetector/templates/index.html [hunk 3]
```diff
@@ -35,7 +35,7 @@
   </center>
   <div class="row">
```
Comment (Rayal-Khatri) [inline Django/weatherdetector/templates/index.html:36]: Test Review matra ho merge garne vye gar reject garne vye ni gar....merge garna siki rakh na vanerw gareko ho

File: Django/weatherdetector/templates/index.html (hunk 2 above)
Comment (sumitdeshar) [reply Django/weatherdetector/templates/index.html:38]: confirmed changes

File: Django/weatherdetector/templates/index.html (hunk 3 above)
Comment (sumitdeshar) [reply Django/weatherdetector/templates/index.html:36]: Fine by me

**Comment by sumitdeshar** [comment]
Test pull sucessfull

File: Django/weatherdetector/weather/views.py [hunk 4]
```diff
@@ -3,5 +3,5 @@
 # Create your views here.
 def index(request):
     if request.method == 'POST': 
-        city = request.POST ['city']
-    return render(request, 'index.html')
\ No newline at end of file
+        city = request.POST['city']
```
Comment (sumitdeshar) [inline Django/weatherdetector/weather/views.py:6]: confirmed changes

**Review by sumitdeshar — COMMENTED** [review]
Reviewed sucessfully

**Pull request merged**